setup.cfg

```
@@ -12,6 +12,7 @@ project_urls =
     Source=https://github.com/twisted/twisted
     Issues=https://twistedmatrix.com/trac/report
     Twitter=https://twitter.com/twistedmatrix
+    Changelog=https://github.com/twisted/twisted/blob/HEAD/NEWS.rst
```

Just asking :) ... but maybe HEAD is better as for GitHub this is the default branch

```suggestion
    Changelog=https://github.com/twisted/twisted/blob/trunk/NEWS.rst
```